modules/administration_manual/pages/configuration/files/encryption/encryption-types.adoc

```diff
@@ -0,0 +1,18 @@
+= Encryption Types
+:page-partial:
+
+ownCloud provides two encryption types:
+
+[cols="40%,60%"]
+|===
+| *User-Key:*
+| Every user has their own private/public key pairs, and the private key is protected by the user’s password.
+
+| *Master Key:*
+| There is only one key (or key pair) and all files are encrypted using that key pair.
+|===
+
+[IMPORTANT]
+====
+These encryption types are *not compatible*.
+====
```

Comment:
I would add a link to (see above)
``configuration/files/encryption/enabling-user-key-encryption.adoc[User-Key Limitations]#limitations``
(attention, the target ``[limitation]`` does  currently not exist, only the header...

Like ``Please see User-Key Limitations for more details``

Reply:
I did add a custom anchor but reviewing the Asciidoctor user manual, it's not necessary. This is because [anchors are created automatically for section titles](https://asciidoctor.org/docs/user-manual/#auto-generated-ids).

Reply:
ok, but I would always add a anchor for two reasons.
- you may change the title but the anchor stays the same for all locations, nothing breakes
- you are consistent in writing anchor + title (a big help for bot that experienced contributors)

just my 2c